package/yast2-registration.changes

```diff
@@ -1,3 +1,10 @@
+-------------------------------------------------------------------
+Wed Feb 23 11:34:03 UTC 2022 - David Diaz <dgonzalez@suse.com>
+
+- Makes the addon selector more "themeable"
+  (gh#yast/yast-registration#566)
```

Comment:
Maybe we should open a bug for this and set it to P2 so it can easier get into SP4...

Reply:
Sure, it's a process thing only, but it should help.